tower-blocking-service/src/lib.rs

```diff
@@ -0,0 +1,26 @@
+use tokio::runtime::current_thread::Runtime;
+use tower_service::Service;
+use tower_service_util::BoxService;
+
+pub struct BlockingSvc<Request, Response, Err> {
+    inner: BoxService<Request, Response, Err>,
```

Comment:
Why does this need to be `BoxService`? I don't think you need anything here besides:

```rust
pub struct Something<T> {
    service: T,
    rt: Runtime,
}
```

Reply:
fixed.